### PR TITLE
Add yolov7-tiny pretrained weights

### PR DESCRIPTION
**Pull request by tkupek**
Weights file has to be added to releases manually.
If necessary, bump release version in the urls.

The weights can be found [here](https://drive.google.com/file/d/1jzFOnNLR70dMrEbjtnexZW33Do4TU2Ft/view?usp=share_link).

### CODE REVIEW

File: yolov7/models/model_configs.py [hunk 1]
```diff
@@ -1175,7 +1175,7 @@ def get_yolov7_tiny_config(num_classes=80, anchor_sizes_per_layer=None, num_chan
     detection_head = Yolov7DetectionHead
 
     return {
-        "state_dict_path": None,
+        "state_dict_path": "https://github.com/Chris-hughes10/Yolov7-training/releases/download/0.1.0/yolov7-tiny_training_state_dict.pt",
```
Comment (Chris-hughes10) [inline yolov7/models/model_configs.py:1178]: I have updated the release with `0.1.0/yolov7-tiny_state_dict.pt`

**Comment by Chris-hughes10** [comment]
Hi @tkupek, thanks for the PR. I have updated the release to include the new state dict, please can you confirm that this works with the updated release and then we can merge

**Comment by tkupek** [comment]
I am not available for the next week, but will update once I'm back.

**Comment by tkupek** [comment]
@Chris-hughes10 the url worked for me. Ready to merge.

**Comment by Chris-hughes10** [comment]
Hi @tkupek, following the discussion in the issue [here](https://github.com/Chris-hughes10/Yolov7-training/issues/12) it seems that a minor code change is required to use the tiny weights. Would you be able to update your PR to reflect this? 

From what I can see, this could be done by setting a flag in the Detection head which controls whether or not to use the implicit modules. For all of the other models, this could default to True, but could be disabled for the tiny weights

**Comment by tkupek** [comment]
@Chris-hughes10 no problem at all, I have already adjusted my own version of the repo.
Pushed the config change just now, please cheack.

File: yolov7/models/model_configs.py (hunk 1 above)
Comment (tkupek) [inline yolov7/models/model_configs.py:1178]:
```suggestion
        "state_dict_path": "https://github.com/Chris-hughes10/Yolov7-training/releases/download/0.1.0/yolov7-tiny_state_dict.pt",
```

[the remark this answers is not in the document]
Comment (tkupek) [reply yolov7/models/model_configs.py]: Sorry for overseeing this. Great hint, we adjusted the implicit modules for the tiny model.

Comment (tkupek) [reply yolov7/models/model_configs.py]: Updated.

**Comment by tkupek** [comment]
Thanks for the hint. The string is updated.

**Pull request merged**